tests/karma/core/components/TestPaginate.js

```diff
@@ -1,123 +1,159 @@
 import React from 'react';
-import { renderIntoDocument, Simulate } from 'react-addons-test-utils';
+import { render } from 'react-dom';
+import { renderIntoDocument } from 'react-addons-test-utils';
+import { Route, Router, createMemoryHistory } from 'react-router';
 import Paginate from 'core/components/Paginate';
 
 describe('<Paginate />', () => {
-  function render({count = 20, currentPage = 1, pager = sinon.spy(), ...extra}) {
-    return renderIntoDocument(
-      <Paginate count={count} currentPage={currentPage} pager={pager} {...extra} />);
-  }
-
-  describe('pageCount()', () => {
-    it('is count / perPage', () => {
-      const root = render({count: 100, perPage: 5});
-      assert.equal(root.pageCount(), 20);
-    });
+  describe('methods', () => {
+    function renderPaginate({count = 20, currentPage = 1, pathname, ...extra}) {
```

Comment:
`renderPagination` would read better to me but I understand if you want to mirror the component name.